sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -1458,4 +1458,14 @@ class SQLQuerySuite extends QueryTest with BeforeAndAfterAll with SQLTestUtils {
       checkAnswer(sql("SELECT * FROM t ORDER BY NULL"), Seq(Row(1, 2), Row(1, 2)))
     }
   }
+
+  test("SPARK-8753: add interval type") {
+    val df = sql("select interval 3 year, interval -14 month, interval 99 second, interval -4 day")
+    checkAnswer(df, Row(36, -14, 99L, -345600L))
+    withTempPath(f => {
```

Comment:
add an inline comment here saying we don't yet support saving out interval data types.